ssm/ssm/projects/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+from django.contrib import admin
+from model_utils.choices import Choices
+
+from ssm.core.models import BaseModel
+from ssm.users.models import User
+
+STATUS = Choices(('started', 'Started'), ('in progress', 'In progress'), ('completed', 'Completed'))
+ROLES = Choices(('project manager', 'Project manager'), ('developer', 'Developer'), ('tester', 'Tester'),
+                ('business analyst', 'Business analyst'), ('front-end developer', 'Front-end developer'),
+                ('back-end developer', 'Back-end developer'), ('data scientist', 'Data scientist'),
+                ('designer', 'Designer'), ('team lead', 'Team lead'), ('product owner', 'Product owner'))
+
+
+class Project(BaseModel):
+    name = models.CharField('Name', max_length=32, unique=True)
+    description = models.TextField('Description', null=True, blank=True)
+    specification = models.FileField('Specification', null=True, blank=True)
```

Comment:
You should store `specification` files in location which will be mapped to docker volume, otherwise you will loose all data after container deletion/recreation, you should be persistent here, also you will have file name collision here if different projects will have the same name for specification, for example `specification.docs`,  so you need to modify model field with `upload_to` attribute, for example, `models.FileField(upload_to='/var/ssm/data/project_{ID}/')`, then create `/var/ssm/data/` in Dockerfile with `RUN mkdir -p /var/ssm/data/` command and map this location to docker volume in `docker-compose-local.yml`, something like this
```
volumes:
  ssm-data:
    driver: local
......
  ssm:
    build:
      context: ../
    volumes:
      - ssm_data:/var/ssm/data/
```